tools/validators/ontology_validator/yamlformat/tests/entity_type_lib_test.py

```diff
@@ -127,9 +127,9 @@ def testEntityTypeUniverseFindsMissingGuids(self):
     findings = types_universe.GetFindings()
```

Comment:
old version should be kept--I don't think my type guid validation changes on github were ever synced to g3